platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -195,4 +212,41 @@ private static void transformation(Source sourceXml, TransformerProperties trans
                     .setContextClassLoader(tccl);
         }
     }
+
+    // This class is used with the processElements method. Pass a function and an instance of
+    // ProcessingResult into the processElements method. Inside the function, set
```

Comment:
This comment doesn't seem to match the code below it. `set use setResult`, does that refer to `setValue`?